File: src/main/java/ru/citeck/idea/completion/js/JsCompletionProvider.java
package ru.citeck.idea.completion.js;

import com.intellij.lang.Language;
import com.intellij.lang.javascript.JavascriptLanguage;
import ru.citeck.idea.completion.CompletionProvider;

public interface JsCompletionProvider extends CompletionProvider {

    @Override
    default Language getLanguage() {
        return JavascriptLanguage.INSTANCE;
    }

}

File: src/main/kotlin/ru/citeck/idea/completion/js/JsCompletionProvider.kt
package ru.citeck.idea.completion.js

import com.intellij.lang.Language
import com.intellij.lang.javascript.JavascriptLanguage
import ru.citeck.idea.completion.contributor.CompletionProvider

interface JsCompletionProvider : CompletionProvider {
    override fun getLanguage(): Language {
        return JavascriptLanguage.INSTANCE
    }
}
